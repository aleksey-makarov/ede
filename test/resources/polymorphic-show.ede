{
  "single": "Lorem ipsum-Phrase",
  "whitespace": "  A Word  ",
  "bool": true,
  "number": 1.0,
  "alpha": ["a", "b", "c", "d", "e"],
  "numeric": [1, 2, 3, 4, 5],
  "object": {"foo": "x", "bar": "y"}
}
---
{{ single | show }}
{{ whitespace | show }}
{{ bool | show }}
{{ number | show }}
{{ alpha | show }}
{{ numeric | show }}
{{ object | show }}
