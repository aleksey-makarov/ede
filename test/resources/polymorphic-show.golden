"Lorem ipsum-Phrase"
"  A Word  "
true
1.0
["a","b","c","d","e"]
[1,2,3,4,5]
{"foo":"x","bar":"y"}
